DOCS-7998 Exchange: Public Portal Login Redesign (#103)
mulesoft/docs-exchange: modules/ROOT/pages/about-portals.adoc

endif::[]
:imagesdir: ../assets/images/

> Exchange public portals are available on the internet to everyone and enable outside developers to access an organization's REST, SOAP, and HTTP APIs.

Reviewer: ```suggestion
Exchange public portals replace what used to be known as _developer portals_ They are now accessible to anyone using the internet, enabling developers outside of an organization to access that organization's REST, SOAP, and HTTP APIs.
```

Reviewer: Later text makes me question what we mean here: anyone on the internet has access to the portal, or only any other organization on Anypoint Platform?

Author: Anyone on the internet has view access. I'll make sure the final wording emphasizes this.

> Exchange portals replace what used to be known as a Developer Portal.

Reviewer: see above

> You can customize the look and content of the portal with a logo, banner image, text, and a favicon for the browser tab.

Reviewer: ```suggestion
You can customize the appearance and the content of the portal by adding a logo, banner image, text, and a favicon for the browser tab.
```
"and" or "or"?

Author: Each one is optional, so I'll use "or."

== Share an Asset or Update the Asset Description

> Exchange Administrators can share an API asset to a public portal.

Reviewer: ```suggestion
As an Exchange administrator, you can edit the description of your organization's API asset so that users outside the organization can identify it, and then share that asset by publishing to a public portal:
```

> . Locate the asset in your organization's Exchange.

Reviewer: ```suggestion
. Locate the asset in your organization's Exchange instance.
```

> . Edit the asset's description, and when done, publish the asset.

Reviewer: ```suggestion
. Edit the asset description and then publish the asset.
```

> . Return to the portal and click the asset. The description updates automatically after changing the description in the asset.

Reviewer: ```suggestion
. Verify the change by returning to the portal and clicking the asset. +
```

> == Users From Other Anypoint Organizations

Reviewer: ```suggestion
== How Users from Other Organizations Access Your Portal
```
Question: when we describe a public portal above, we say that anyone on the internet can use it; now we are qualifying this to include only users from other organizations using Anypoint Platform?

Author: Anyone on the internet can view the portal. Users from other Anypoint Platform organizations can log in to access additional features. I'll update the wording to fit.

Users can view a public portal without logging in, and must log in to use features such as reviewing assets, downloading assets, creating a client application to request API access, and accessing an API.

Users from other Anypoint Platform organizations can use your public portal to register client applications so they can access your APIs. Because they do not belong to your organization, they are external users of your organization's portal.

> An organization administrator can see only the basic profile of an external user, which contains only this information:

Reviewer: ...so, do we mean this:

Although anyone on the internet can view your public portal, only someone who can log in to it can use portal features. These _external users_ are users from organizations on Anypoint Platform outside of your own who use your public portal to register client applications and access your APIs. As an organization administrator, you can see only their basic profiles, which include:

Reviewer: However, we have not said how these users are able to log in, or even mentioned logging in prior to this.

Author: Yes, that's correct. These users can log in by clicking the "Login" link on each page, and I'll update to reflect this.

> * Name

Reviewer: ```suggestion
* Personal name
```

* Username
* Email
* Primary organization name

> The first time an external user logs in to your public portal, the portal requests permission for read-only access to this basic profile. When the user approves the request, they are added in the Anypoint Platform tab *Access Management* > *Users* > *External Users*. As your organization's administrator, you can enable or disable each external user on that tab. Disabled external users cannot sign in to your public portal.

Reviewer: ```suggestion
The first time someone attempts to log in to your public portal, the portal requests permission for read-only access to this basic profile. When the person trying to log in approves this request, their information is added to the *External Users* section of *Users* on the *Access Management* tab in Anypoint Platform. However, they cannot log in until you, as your organization's administrator, enable them as an external user on that tab. If you disable them, they cannot sign in to your public portal.
```
A little unclear here: does the admin have to actively disable someone added as an external user? Does this persons info persist on record as either enabled or disabled? or does it disappear if NOT enabled? Reword to clarify.

> To grant an external user permission to perform tasks such as deploying an application, invite the user to join your organization.

Reviewer: Is this the final step to ENABLING a user: an official invitation?
Or is this a level of access that is additional to being logged in?
Or is this how someone on the internet initiates going from viewing to being able to log in?
Reword to clarify.

Author: This is an additional level of access. I'll reword to clarify.
